desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -23,6 +24,7 @@ GenericName[ja]=Flashプレイヤー
 GenericName[ko]=플래시 플레이어
 GenericName[nl]=Flash-speler
 GenericName[pl]=Odtwarzacz Flash
+GenericName[pt]=Flash Player
```

Comment:
I might become a Portuguese linguist after these reviews, but I'll bet 5€ it should be _Reprodutor Flash_ here.

Reply:
This got changed to "Leitor" since, as explained in https://github.com/ruffle-rs/ruffle/pull/18426#issuecomment-2452470579.

Reply:
That comment refers to `pt_PT`, and I believe it will be "reprodutor" in `pt_BR` :eyes: